src/theory/output_channel.h

```diff
@@ -55,41 +51,37 @@ class LemmaStatus {
    */
   unsigned getLevel() const throw() { return d_level; }
 
-};/* class LemmaStatus */
+ private:
+  Node d_rewrittenLemma;
+  unsigned d_level;
+}; /* class LemmaStatus */
 
 /**
  * Generic "theory output channel" interface.
  */
 class OutputChannel {
-  /** Disallow copying: private constructor */
-  OutputChannel(const OutputChannel&) CVC4_UNDEFINED;
-
-  /** Disallow assignment: private operator=() */
-  OutputChannel& operator=(const OutputChannel&) CVC4_UNDEFINED;
-
-public:
-
-  /**
-   * Construct an OutputChannel.
-   */
-  OutputChannel() {
-  }
+ public:
+  /** Construct an OutputChannel. */
+  OutputChannel() {}
 
   /**
    * Destructs an OutputChannel.  This implementation does nothing,
    * but we need a virtual destructor for safety in case subclasses
    * have a destructor.
    */
-  virtual ~OutputChannel() {
-  }
+  virtual ~OutputChannel() {}
+
+  /** Disallow copying: private constructor */
+  OutputChannel(const OutputChannel&) = delete;
+
+  /** Disallow assignment: private operator=() */
+  OutputChannel& operator=(const OutputChannel&) = delete;
 
   /**
    * With safePoint(), the theory signals that it is at a safe point
    * and can be interrupted.
```

Comment:
Maybe add a comment about the interrupted exceptions that can be thrown?

Reply:
Done.